ingest.py

```
@@ -0,0 +1,65 @@
+        return ""
+
+def process_pdf(file_path):
+    """Lee un archivo .pdf y extrae el texto de cada página."""
+    try:
+        doc = fitz.open(file_path)
+        return "\n".join([page.get_text() for page in doc])
+    except Exception as e:
+        print(f"Error procesando {file_path}: {e}")
+        return ""
+
+def process_csv(file_path):
```

Ruff (0.12.2)

Missing return type annotation for public function `process_csv`

(ANN201)